public/expressions/overlay_anomalies.ts

```diff
@@ -79,13 +79,13 @@ const getAnomalies = async (
   return parsePureAnomalies(anomalySummaryResponse);
 };
 
-const getDetectorResponse = async (detectorId: string) => {
+export const getDetectorResponse = async (detectorId: string) => {
```

Comment:
nit: you could pull out all of these helper fns used in the expression fn into a separate file and call them in here, to keep the function file a little cleaner and more concise. This is what we do on OSD side. But no strong feelings either way, i'm fine with the current way too